src/validation-errors/additional-prop.js

```diff
@@ -10,8 +10,8 @@ export default class AdditionalPropValidationError extends BaseValidationError {
     output.push(chalk`{red {bold ADDTIONAL PROPERTY} ${message}}\n`);
 
     return output.concat(
-      printJson(data, `${dataPath}/${params.additionalProperty}`)(() => {
-        return chalk`😲  {bold ${params.additionalProperty}} is not expected here!`;
+      printJson(data, `${dataPath}/${params.additionalProperty}`, { indent: this.indent })(() => {
```

Comment:
So, `indent` for testing?

Reply:
Not sure if I got it. I just pass indent as an option here.